string name
sensor_msgs/Image image
float32 meter_per_pixel
---